traffic_ops/traffic_ops_golang/deliveryservice/safe.go

```diff
@@ -0,0 +1,124 @@
+package deliveryservice
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import (
+	"database/sql"
+	"fmt"
+	"net/http"
+
+	"github.com/apache/trafficcontrol/lib/go-tc"
+	"github.com/apache/trafficcontrol/traffic_ops/traffic_ops_golang/api"
+	"github.com/apache/trafficcontrol/traffic_ops/traffic_ops_golang/tenant"
+)
+
+const safeUpdateQuery = `
+UPDATE deliveryservice
+SET display_name=$1,
+    info_url=$2,
+    long_desc=$3,
+    long_desc_1=$4
+WHERE id = $5
+RETURNING id
```

Comment:
nit but need to return id ???

Reply:
I'm not 100% sure about this because I copy/pasted this from Rob's PR, but I _think_ if it doesn't return _something_ then the `RowsAffected` won't be accurate.

Reply:
I _dont_ think that is true but its not hurting anything at all so 🤷‍♂

Reply:
I think `RowsEffected` is fine without it. I think it's an artifact of the "CRUDer" framework needing it. Probably copied from a CRUDer query, or maybe my old PR used the CRUDer stuff, I don't remember. But yeah, not really hurting anything.